example/kokkos_pi/kokkos_pi.cpp

```diff
@@ -65,6 +65,8 @@ using namespace parthenon::driver::prelude;
 
 using View2D = Kokkos::View<Real **, Kokkos::LayoutRight, Kokkos::DefaultExecutionSpace>;
 
+const int nghost = 2;
```

Comment:
Should this be set in the input file and `Globals::ghost` used in the file?

Reply:
I actually tried to use `Globals::nghost` and something prevented it from working.  Honestly, I just didn't think it was worth it to track down the issue.  This example doesn't really actually use the parthenon machinery anyway.  I've even been thinking lately (after seeing it trip some people up) that we should remove it from the examples that ship with parthenon.